runtime/queries/metricsview_timeseries.go

```diff
@@ -90,7 +90,8 @@ func (q *MetricsViewTimeSeries) Resolve(ctx context.Context, rt *runtime.Runtime
 }
 
 func (q *MetricsViewTimeSeries) buildMetricsTimeSeriesSQL(mv *runtimev1.MetricsView) (string, []any, error) {
-	timeCol := fmt.Sprintf("DATE_TRUNC('%s', %s) AS %s", q.TimeGranularity, mv.TimeDimension, mv.TimeDimension)
+	timestampColumnName := safeName(mv.TimeDimension)
+	timeCol := fmt.Sprintf("DATE_TRUNC('%s', %s) AS %s", q.TimeGranularity, timestampColumnName, timestampColumnName)
 	selectCols := []string{timeCol}
```

Comment:
What if `mv.TimeDimension == ""`?

Reply:
Also, `timeCol` and `timestampColumnName` sound like the same thing. In Google's Go style guide, check out the "Variable names" section [here](https://google.github.io/styleguide/go/decisions#variable-names). Maybe `timeExpr` and `timeCol`?

Reply:
Just reading the link you posted:

> Naming is more art than science.